be/src/http/action/http_stream.cpp

```diff
@@ -245,13 +245,13 @@ void HttpStreamAction::on_chunk_data(HttpRequest* req) {
             if (ctx->schema_buffer->pos + remove_bytes < config::stream_tvf_buffer_size) {
                 ctx->schema_buffer->put_bytes(bb->ptr, remove_bytes);
             } else {
-                ctx->need_schema = true;
+                LOG(INFO) << "use a portion of data to request fe to obtain column information";
                 ctx->is_read_schema = false;
```

Comment:
Does is_read_schema need to be a member in ctx???   I think it could just be a local variable in this function.